### ISSUE TITLE
gitconfig highlight broken

### ISSUE THREAD

**Issue by thorade** [atom/language-git#12]
The syntax highlihgt for the following Git Config file looks somehow broken (screenshot below).
Using the language-ini package it looks better.

```ini
[merge]
	tool = kdiff3
[mergetool "kdiff3"]
	path = C:/Program Files (x86)/KDiff3/kdiff3.exe
[diff]
	guitool = kdiff3
[difftool "kdiff3"]
	path = C:/Program Files (x86)/KDiff3/kdiff3.exe
[core]
	editor = \"C:/Program Files (x86)/GitExtensions/GitExtensions.exe\" fileeditor
	autocrlf = True
[credential]
[filter "lfs"]
	clean = git-lfs clean %f
	smudge = git-lfs smudge %f
	required = true
```

![atom_gitconfig](https://cloud.githubusercontent.com/assets/314571/10764061/bd826c64-7ccc-11e5-8aeb-78484028a50b.png)

**Comment by 50Wliu** [comment]
This is actually a duplicate of #4 and will be fixed by #14.

**Comment by thorade** [comment]
Thanks!
